src/theme/components/tabs.ts

```diff
@@ -0,0 +1,51 @@
+const CustomTabs = {
+  baseStyle: {
+    tablist: {
+      borderColor: "ui.black",
+      maxWidth: {
```

Comment:
Note how you can add responsive styles by passing in an object rather than a single value for any specific CSS rule. In this case, "base" corresponds to the native mobile viewport and "lg" corresponds to "960px". Chakra does `min-width` by default for mobile-first responsive design.

It's also possible to pass in something like `maxWidth: ["70%", null, "100%"]` but I prefer to use the responsive object to be clearer about the breakpoints, and it's easier to read.

Reply:
This is, obviously, very important.  We should talk about this more and make sure all of the DS devs are aware of how to handle responsive styles.